index.html

```diff
@@ -0,0 +1,59 @@
+  </head>
+
+  <body>
+
+    <nav id="navbar">
+    <ul>
+      <li><a href="#welcome-section" >Welcome Section</a></li>
+      <li><a href="#projects" >Projects</a></li>
+    </ul>
+
+    </nav>
+
+    <section id="welcome-section">
+        <h1>Welcome to the portfolio</h1>
+        <div class="welcome">
+          <h2>Welcome to the portfolio</h2>
+          <h3>The best web developer</h3>
+      </div>
+
+    </section>
+
+
+    <section id="projects" class="grid-wrapper">
+      <div class="project-tile"><img src="https://media.istockphoto.com/photos/first-person-point-of-view-of-a-woman-paddling-on-a-stand-up-paddle-picture-id1288844330?b=1&k=20&m=1288844330&s=170667a&w=0&h=nckXG0H5kPDbgDpC8iTObsiqG7Jwt6CeLuJ2WxdOTp4="><a href="https://google.com">Link to the Project</a></div>
+      <div class="project-tile"><img src="https://media.istockphoto.com/photos/first-person-point-of-view-of-a-woman-paddling-on-a-stand-up-paddle-picture-id1288844330?b=1&k=20&m=1288844330&s=170667a&w=0&h=nckXG0H5kPDbgDpC8iTObsiqG7Jwt6CeLuJ2WxdOTp4="><a href="https://google.com">Link to the Project</a></div>
+      <div class="project-tile"><img src="https://media.istockphoto.com/photos/first-person-point-of-view-of-a-woman-paddling-on-a-stand-up-paddle-picture-id1288844330?b=1&k=20&m=1288844330&s=170667a&w=0&h=nckXG0H5kPDbgDpC8iTObsiqG7Jwt6CeLuJ2WxdOTp4="><a href="https://google.com">Link to the Project</a></div>
+      <div class="project-tile"><img src="https://media.istockphoto.com/photos/first-person-point-of-view-of-a-woman-paddling-on-a-stand-up-paddle-picture-id1288844330?b=1&k=20&m=1288844330&s=170667a&w=0&h=nckXG0H5kPDbgDpC8iTObsiqG7Jwt6CeLuJ2WxdOTp4="><a href="https://google.com">Link to the Project</a></div>
+      <div class="project-tile"><img src="https://media.istockphoto.com/photos/first-person-point-of-view-of-a-woman-paddling-on-a-stand-up-paddle-picture-id1288844330?b=1&k=20&m=1288844330&s=170667a&w=0&h=nckXG0H5kPDbgDpC8iTObsiqG7Jwt6CeLuJ2WxdOTp4="><a href="https://google.com">Link to the Project</a></div>
+      <div class="project-tile"><img src="https://media.istockphoto.com/photos/first-person-point-of-view-of-a-woman-paddling-on-a-stand-up-paddle-picture-id1288844330?b=1&k=20&m=1288844330&s=170667a&w=0&h=nckXG0H5kPDbgDpC8iTObsiqG7Jwt6CeLuJ2WxdOTp4="><a href="https://google.com">Link to the Project</a></div>
+      <div class="project-tile"><img src="https://media.istockphoto.com/photos/first-person-point-of-view-of-a-woman-paddling-on-a-stand-up-paddle-picture-id1288844330?b=1&k=20&m=1288844330&s=170667a&w=0&h=nckXG0H5kPDbgDpC8iTObsiqG7Jwt6CeLuJ2WxdOTp4="><a href="https://google.com">Link to the Project</a></div>
+      <div class="project-tile"><img src="https://media.istockphoto.com/photos/first-person-point-of-view-of-a-woman-paddling-on-a-stand-up-paddle-picture-id1288844330?b=1&k=20&m=1288844330&s=170667a&w=0&h=nckXG0H5kPDbgDpC8iTObsiqG7Jwt6CeLuJ2WxdOTp4="><a href="https://google.com">Link to the Project</a></div>
+    </section>
```

Comment:
I suggest using semantic tags like header, main, etc. It will help you with the relevance of each part of your page.